activesupport/lib/active_support/error_reporter.rb

```diff
@@ -31,10 +31,11 @@ module ActiveSupport
   # +severity+ can be one of +:error+, +:warning+ or +:info+. Handled errors default to the +:warning+
   # severity, and unhandled ones to +error+.
   #
-  # Both `handle` and `record` pass through the return value from the block. In the special case of `handle` handling an
-  # error, a fallback value can be provided that will be returned:
+  # Both `handle` and `record` pass through the return value from the block. In the case of `handle`
```

Comment:
That one is my fault:
```suggestion
  # Both +handle+ and +record+ pass through the return value from the block. In the case of +handle+
```

I'm a bit too used to markdown...

Reply:
Oh and I absolutely copied that from you.